src/main/java/com/ebivariation/contigalias/repo/AssemblyRepository.java

```diff
@@ -20,6 +20,8 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import javax.swing.text.html.Option;
```

Comment:
I think you don't need this import

Reply:
Removed. I think it got auto imported when I mistyped Optional as Option at some point.